task/jira-get-ticket-status/0.1/jira-get-ticket-status.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: jira-get-ticket-status
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
```

Comment:
latest version of pipelines is `0.59`, since this is a new task, perhaps we can keep a near latest version for this.

Reply:
done!